### ISSUE TITLE
Feature Request: MariaDB possibility instead of postgres

### ISSUE THREAD

**Issue by Alulu2401** [linkwarden/linkwarden#546]
hello guys,

I'm really interested in using linkwarden for my personnal workflow. 
i have an instance of mariaDB runing in my workshop and i'd really like to use linkwarden with it as database.

So the request is simple if i'm not alone in this case : provide mariadb compatibility :D 

Thanks a lot.
Olivier

**Comment by dada051** [comment]
Yes could be great if I don't need to setup Postgres on my home lab

**Comment by Nekotje** [comment]
Would be great to be able to connect to an excising mariadb database
